client/src/components/BlogPostCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { FaLocationDot } from 'react-icons/fa6';
+import { PiHeartFill } from 'react-icons/pi';
+import { Link } from 'react-router-dom';
+import { Entry } from '../lib/types';
+
+type props = {
+  entry: Entry;
+};
+
+export default function BlogPostCard({ entry }: props) {
+  const [underline, setUnderline] = useState(false);
+
+  function handleUnderline() {
+    setUnderline(!underline);
+  }
+
+  return (
+    <div className="h-48 border-b border-black mt-3">
+      <div className="flex items-center justify-between py-1">
+        <div className="flex items-center">
+          <span>
+            Article by{' '}
+            <span className="underline cursor-pointer">Tim Silva</span>
```

Comment:
I assume that eventually this will be driven from the database?

Reply:
Yes